server/flyway/sql/V39__add_client_role.sql

```diff
@@ -0,0 +1,47 @@
+-- Delete any role assignments to the CLIENT initial testing roles
+DELETE FROM app_fam.fam_user_role_xref WHERE role_id IN
+    (SELECT role_id from app_fam.fam_role WHERE application_id IN
+        (SELECT application_id FROM app_fam.fam_application WHERE application_name IN ('CLIENT_DEV', 'CLIENT_TEST', 'CLIENT_PROD'))
+    )
+;
+
+-- Delete previously created CLIENT testing roles for client_dev, client_test and client_prod applications
+DELETE FROM app_fam.fam_role WHERE application_id IN (
+    SELECT application_id FROM app_fam.fam_application WHERE application_name IN ('CLIENT_DEV', 'CLIENT_TEST', 'CLIENT_PROD')
+);
```

Comment:
The two delete sql code are coping from v24, feel the logic makes sense to cleanup any user role assignment and then cleanup the role